apps-rendering/src/components/liveBlocks.tsx

```diff
@@ -43,33 +23,19 @@ const LiveBlocks: FC<LiveBlocksProps> = ({ blocks, format }) => {
 			{blocks.map((block) => {
 				// TODO: get page number
 				const blockLink = `${1}#block-${block.id}`;
+				const blockFirstPublished = block.firstPublished.kind === OptionKind.Some
```

Comment:
This may be unnecessary if this changes to a `Date` (see below), but an alternative way to write this is:

```ts
pipe(
    block.firstPublished,
    map(Number),
    withDefault<number | undefined>(undefined),
)
```

Either are fine, I think it's up to you which you prefer 🙂. If this does change to a `Date`, the conversion from `Option` is simpler (and we should probably add it to `lib` if we're going to be doing it more often 🤔):

```ts
withDefault<Date | undefined>(undefined)(block.firstPublished)
```

In fact maybe we should anyway:

```ts
const toNullable = <A>(opt: Option<A>): A | undefined =>
	withDefault<A | undefined>(undefined)(opt)
```

So the original `pipe` becomes:

```ts
pipe(
    block.firstPublished,
    map(Number),
    toNullable,
)
```

Reply:
Thanks Jamie, I have had a play around with this. It certainly seems neater and more in keeping with AR to write it as above, so I have changed it. I can see your point about date but I'm less clear about how you intended that to be implemented.